index.html

```diff
@@ -358,6 +350,15 @@
 <li>Let <var>deadline</var> be a time in the future until which the browser expects to remain idle.</li>
 <li>If <var>deadline</var> - <var>now</var> is greater than 50ms, then cap <var>deadline</var> by setting it to
 be <var>now</var> + 50ms.</li>
+
+<!-- TODO
+  (1) What's the right plumbing below? "document's Window object's list..."?
+  - Should we instead be talking about browsing contexts associated with the event loop?
```

Comment:
"document's Window object" seems correct. Browsing contexts does not seem correct.

In practice Window seems like the right place to associate things, since you want the idle callbacks to be cleared upon navigation. If they were associated with the browsing context, or with the event loop, they would survive even after navigation.